portal/forms/play.py

```diff
@@ -56,9 +56,11 @@ class StudentLoginForm(forms.Form):
         label='Password',
         widget=forms.PasswordInput)
 
+    view_options = {'is_recaptcha_valid': False, 'is_recaptcha_visible': False}
+
     def clean(self):
-        if self.has_error('recaptcha'):
-            raise forms.ValidationError("Invalid name, class access code, password or captcha")
+        if self.view_options['is_recaptcha_visible'] and not self.view_options['is_recaptcha_valid']:
+            raise forms.ValidationError('Incorrect email address, password or captcha')
```

Comment:
Since the student login form requires fields like name, class access code and password, I think we'd want to keep the previous error message: `"Invalid name, class access code, password or captcha"` - since students don't have an email address.